src/pages/functional-testing-framework/test-writing/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Test writing | Commerce Testing
+description: Learn about Functional Testing Framework test writing principles to optimize your Adobe Commerce and Magento Open Source test plans.
+---
+
+# Test writing
```

Comment:
You need to put something in this topic, since it must exist.

Reply:
Says who?! Just kidding. I'll add something.